requirements.txt

```diff
@@ -0,0 +1,11 @@
+cryptography==2.3.1
```

Comment:
Pipenv doesn't really work for me and python setup.py install fails for me, because of the pygerrit installation fails with PBR error, so I wanted an alternative. But the PBR error can be worked around by setting `PBR_VERSION=1.0.1-dev1` before running `python setup.py install` so I can remove the requirements.txt I guess. Just pointing out why I wanted to have one